cmake/Modules/GoCompile.cmake

```diff
@@ -27,6 +27,9 @@ if(GO_BUILD)
   endif()
 endif()
 list(APPEND args "-tags" "integration")
+if (WIN32 AND GO_WIN_UI)
+  list(APPEND args "-ldflags" "-H windowsgui")
```

Comment:
> Aren't those flags to the C compiler and linker, not the go linker? 

You're probably right.

Please add a comment so that people don't have to search in vain like I did.